src/lib/server/database.ts

```diff
@@ -75,8 +75,9 @@ client.on("open", () => {
 	messageEvents.createIndex({ createdAt: 1 }, { expireAfterSeconds: 60 }).catch(console.error);
 	sessions.createIndex({ expiresAt: 1 }, { expireAfterSeconds: 0 }).catch(console.error);
 	sessions.createIndex({ sessionId: 1 }, { unique: true }).catch(console.error);
-	assistants.createIndex({ createdBy: 1 }).catch(console.error);
+	assistants.createIndex({ createdById: 1, userCount: -1 }).catch(console.error);
 	assistants.createIndex({ userCount: 1 }).catch(console.error);
```

Comment:
with the newly added indices, I don't think we need this index. Do we? And even if we need it, we should change it to `-1` ? cc: @coyotte508 @nsarrazin

Reply:
I'm talking about this one
```ts
assistants.createIndex({ userCount: 1 }).catch(console.error);
```

Reply:
If the query can be just `{userCount: {$gt: 1}}`, then we should keep it.

`-1` or `1` doesn't have a difference in indexes with only one key (mongo can always "reverse" the index). In compound indexes, it makes a difference

Reply:
yes, query can be just `{userCount: {$gt: 1}}`. Therefore, keeping it